SimpleSession.py

```diff
@@ -109,10 +120,42 @@ def save(self, session):
         with open(session, 'w') as sess_file:
             json.dump(data, sess_file, indent=4)
 
+    def input_changed(self, session_name_prefix):
+        """
+            on input changed open autocomplete menu with a delay
+        """
+        if len(session_name_prefix) > 0 and \
+        self.input_panel and \
+        self.input_panel.command_history(0)[0] not in ['insert_completion', 'insert_best_completion']: #remove the looping
+            if self.input_panel.is_auto_complete_visible():
+                self.input_panel.run_command('hide_auto_complete')
+            delay = 500
+            sublime.set_timeout(lambda: self.run_autocomplete(), delay)
+        else:
+            return
+
+    def run_autocomplete(self):
+        self.input_panel.run_command('auto_complete')
+
+    def on_query_completions(self, prefix, locations):
+        if len(prefix) > 0:
+            completions_list = getSessionFileNames()
+            #needed the "hit Tab" label due to https://github.com/SublimeTextIssues/Core/issues/1727            
+            completions_list = [["{0}\t hit Tab to insert".format(item), item] for item in completions_list if item.startswith(prefix)]
+            if len(completions_list) == 1 and completions_list[0][1] != prefix:
+                completions_list += [["{0}\t hit Tab to insert".format(prefix), prefix]]
+            return (
+                        completions_list,
+                        sublime.INHIBIT_WORD_COMPLETIONS | sublime.INHIBIT_EXPLICIT_COMPLETIONS
+                    )
+        else: #if no prefix return empty list
+            return
```

Comment:
The comment here says "return empty list," but this returns `None`. I don't know which is correct, but at least one of these lines should probably be updated.

Reply:
Mistyped in comment